.ruff.toml

```diff
@@ -0,0 +1,5 @@
+target-version = "py310"
+line-length = 99
```

Comment:
I was just about to say we should add this. Nice :)

Reply:
Should this be py311? cf. https://github.com/key4hep/k4geo/pull/398/files#diff-b3236d4cb260ca6acce083493fcdc42816f732fc3d5aefb263a3c3fe9db4f81f

Reply:
I tried to be compatible with the black config from k4FWCore https://github.com/key4hep/k4FWCore/blob/main/pyproject.toml

Reply:
Then let's put py310 in k4geo (@tmadlener)?

Reply:
Updating to `py311` here and in k4FWCore and reformatting doesn't change any files. We have python 3.11 in the nightlies. I think we could just update and have py311 everywhere

I can also copy the extra formatting options from k4geo (it's just being explicitly setting the values same as the default right?)

Reply:
> it's just being explicitly setting the values same as the default right?

Yes. Specifically the defaults that make ruff work / format the same as black.

Reply:
> I think we could just update and have py311 everywhere

I would be in favor of that.

Reply:
Updated here